cypress/features/regression/test/actionPopoverNoIFrame.feature

```diff
@@ -2,7 +2,7 @@ Feature: Action Popover component in noIFrame
   I want to change Action Popover component properties in noIFrame
 
   Background: Open Action Popover component page in noIFrame
-    Given I open "Test Action Popover" component in iframe
+    Given I open "Test Action Popover" component in noiFrame
```

Comment:
try to use step which we have for tests in `test` folder and open in noIframe
`Given('I open basic Test {string} component page in noIframe', (component) => {
  visitComponentUrl(component, 'basic', true, 'test-');
});`

Reply:
I can't use that step, because the suffix for the Action Popover is `--default`